src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -48,8 +48,8 @@ const workspaceById = gql`
         queries {
           id
           title
-          cql
```

Comment:
I'm removing this field because it doesn't seem to be used elsewhere in this file